github/Repository.py

```diff
@@ -1993,15 +1993,18 @@ def get_assignees(self) -> PaginatedList[NamedUser]:
         """
         return PaginatedList(github.NamedUser.NamedUser, self._requester, f"{self.url}/assignees", None)
 
-    def get_branch(self, branch: str) -> Branch:
+    def get_branch(self, branch: str, lazy: Opt[bool] = NotSet) -> Branch:
```

Comment:
Why add this additional argument? Can't the end-user decide this by calling the additional method afterwards? Why do they need to specify it here?